source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,320 @@ Sorts Builders
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+
+Sorting Direction
+-----------------
+
+The ``Sorts`` class provides two methods for specifying the sort order of data
+that does not contain any duplicate values. Data can be sorted from smallest to largest 
+with ``Sorts.ascending()``, and data can be sorted from largest to smallest with
+``Sorts.descending()``. The following two subsections provide examples
+on how to use these two methods. 
+
+Ascending
+~~~~~~~~~
+
+To specify an ascending sort, use the ``Sorts.ascending()``
+static factory method. An ascending sort returns 
+your documents from smallest to largest on the field name specified.
+
+The ``ascending`` method can be used as follows:
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts
+
+   // <MongoCollection setup code here>
+
+   collection.find().sort(Sorts.ascending("<field name>"))
+
+This will return a 
+:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html>`
+object containing the documents in your collection sorted 
+from smallest to largest on the specified field name. 
+
+.. example::
+
+   In this code example, we use ``ascending`` to
+   sort the ``sort_example`` collection by ``"_id"``:
+
+   .. code-block:: java
+   
+      import static com.mongodb.client.model.Sorts.ascending;
+   
+      // <MongoCollection setup code here>
+      
+      Bson id_sort = ascending("_id");
+      List<Document> results = new ArrayList<>();
+      collection.find().sort(id_sort).into(results);
+      for (Document result : results) {
+          System.out.println(result.toJson());
+      }
+
+   The output of the above code snippet should look something like this: 
+   
+   .. code-block:: none
+   
+      {"_id": 1, "letter": "c", "food": "coffee with milk"}
+      {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+      {"_id": 3, "letter": "a", "food": "maple syrup"}
+      ...
+   
+An important thing for you to keep in mind is that MongoDB does
+not guarantee sort order for documents that have fields with identical values.
+This means that ``ascending("letter")`` could have any of these
+documents in the first position: 
+
+.. code-block:: none
+
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 2, "letter": "a", "donuts and coffee"}
+
+If you would like to have a guaranteed sort order 
+for documents that have fields with identical values
+you can specify additional fields to sort
+on in the event of a tie. 
+
+We can specify an ``ascending`` sort on ``"letter"`` followed
+by ``"_id"`` as follows:
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(ascending("letter", "_id"))
+
+The above code snippet returns the documents in the
+``sort_example`` collection in the following order: 
```

Comment:
Consider linking back to the example documents when mentioning this collection since the page is fairly long.

Reply:
User might not know what the sort example collection is. If they use TOC they could miss it. Add reference to the collection and turn all instances of "sort_example" into links to that reference. 

On most pages you can't see it at the same time as references to it. 

Alternatively, you could say 'at the top of the page', but linking is probably no harder and better.

Reply:
Thanks for the note. Fix: https://github.com/mongodb/docs-java/pull/59/commits/38f50339a909443d6af8583a0e6e702610fd154f

Going forward, I'll try to be more mindful of vertical scrolling a user might have to do to make sense on an example. I will also try to be mindful of how users can go outside the expected flow of a page (missing sort_example definition).